code/modules/mob/living/carbon/carbon.dm

```diff
@@ -610,6 +610,13 @@
 
 	return null
 
+/mob/living/carbon/proc/isTeleViewing(var/client_eye)

```

Comment:
No, why is it a proc at the carbon level?
Why would only a carbon mob ever use this?